src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -26,31 +29,119 @@ class HoldRequest extends React.Component {
     this.setState({ data: Store.getState() });
   }
 
+  /**
+   * requireUser()
+   * Redirectes the patron to OAuth log in page if he/she hasn't been logged in yet.
+   *
+   * @return {Boolean}
+   */
   requireUser() {
-    if (!this.state.patron || !this.state.patron.id.length) {
-      const fullUrl = encodeURIComponent(window.location.href);
-      window.location.replace(`${config.loginUrl}?redirect_uri=${fullUrl}`);
-      return false;
+    if (this.state.patron && this.state.patron.id) {
+      return true;
     }
-    return true;
+
+    const fullUrl = encodeURIComponent(window.location.href);
+
+    window.location.replace(`${config.loginUrl}?redirect_uri=${fullUrl}`);
+
+    return false;
   }
 
-  render() {
-    const searchKeywords = this.props.searchKeywords;
-    const record = this.props.bib;
-    const title = record.title[0];
-    const bibId = record['@id'].substring(4);
-    const itemId = this.props.params.id;
-    const selectedItem = LibraryItem.getItem(record, itemId);
-    const location = LibraryItem.getLocation(record, itemId);
+  /**
+   * renderLoggedInInstruction(patronName)
+   * Renders the HTML elements and contents based on the patron data
+   *
+   * @param {String} patronName
+   * @return {HTML Element}
+   */
+  renderLoggedInInstruction(patronName) {
+    return (patronName) ?
+      <p className="loggedInInstruction">You are currently logged in as <strong>{patronName}</strong>. If this is not you, please <a href="https://isso.nypl.org/auth/logout">Log out</a> and sign in using your library card.</p>
```

Comment:
The original one is uppercase. I think we can update it until the content is decided?